docs/customization/controller.rst

```diff
@@ -10,7 +10,8 @@ If you want to override a controller action, check which controller you should b
     There are two types of controllers we can define in Sylius:
 
     **Resource Controllers** - are based only on one Entity, so they return only the resources they have in their name. For instance a ``ProductController`` should return only products.
-    **Standard Controllers** - non-resource; these may use many entities at once, they are useful on more general pages. We are extending these controllers only if the actions we want cannot be done through yaml configuration - like sending emails.
+    **Standard Controllers** - non-resource; these may use many entities at once, they are useful on more general pages.
+    We are extending these controllers only if the actions we want cannot be done through yaml configuration - like sending emails.
```

Comment:
`extending` -> `defining`. We don't used to extend any class at all in standard controllers.